### ISSUE TITLE
Benchmark async resolver parallelization 

### ISSUE THREAD

**Issue by theferrit32** [clingen-data-model/genegraph#150]
The goal of this issue is to perform a benchmark of converting one or more resolvers to be async, to gauge the effectiveness and speedup potential for doing a more extensive parallelization of resolvers. Primary candidates for parallelization are field resolvers on graphql listing endpoints. Threads should be pulled from a size-capped pool to avoid thread creation/cleanup overhead and to prevent an explosion of threads on graphql listing endpoints.

https://lacinia.readthedocs.io/en/latest/resolve/async.html

**Comment by theferrit32** [comment]
Note that this issue and https://github.com/clingen-data-model/genegraph/issues/4 would each likely significantly mitigate turnaround time for listing queries and are not mutually exclusive. #4 differs from parallelization like here because it would reduce I/O roundtrips themselves, not just perform the roundtrips in parallel. However it is unlikely that all fields could be or would be desired to be included in #4, so parallelizing those not retrieved in the initial database read (as described in this ticket) would still be beneficial.

**Comment by KelseaChang5** [comment]
We're putting this on hold until we see that this is worth spending time on if users are experience performance problems
